### DISCUSSION TITLE
Release v0.80.0 : Grid view & Form view

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#704] [Features]
<h1 align="center" style="border-bottom: none">
    <b>
        <a href="https://www.nocodb.com">NocoDB</a><br>
    </b>
    🎃 The Open Source Airtable Alternative 🎃 <br>
</h1>

<p align="center">
Turns any MySQL, PostgreSQL, SQL Server, SQLite & MariaDB into a smart-spreadsheet.
</p>
<p align="center">
    <a href="http://www.nocodb.com"><b>Website</b></a> •
    <a href="https://discord.gg/5RgZmkW"><b>Discord</b></a> •
    <a href="https://twitter.com/nocodb"><b>Twitter</b></a> •
    <a href="https://github.com/nocodb/nocodb"><b>GitHub</b></a> •
    <a href="https://docs.nocodb.com/"><b>Documentation</b></a>
</p>

---------------------------------------------------------------------

# [v0.80.0](https://github.com/nocodb/nocodb/releases/tag/0.80.0) Release note: FORM & GRID view
Another step towards better team collaboration!
With Noco VIEWs, Share & collect customized data that you wish, the way you want!

## GRID Views
```**GRID views**``` are great way of sharing information sub-set to different team/people. Shared grid views are READ only. Your team will get information access that they need- without compromising on complete database access.

### Features
- Customize table data into different views depending on the team/ audience
  - Show/ hide columns
  - Filter rows
  - Sort/order contents
- Create/ share any number of unique grid views
- Copy existing view as base for a new view, instead of always starting afresh
- Protect shared view access using a Password

### GRID View: How to Create, Customize & Share?
![Grid](https://user-images.githubusercontent.com/86527202/138726069-226dc0e6-4064-49d8-8a57-57ddd32009c3.gif)

------------------------------------------

## FORM Views
```**FORM views**``` provide easy access to gather information from different people, record them in database and are easy to create using existing fields of database table. 

### Features
- Customize form view with ease- drag/drop fields that you need
- Validation supported based on field datatype
- Create/ share any number of unique form views
- Protect shared view access using a Password
- Copy existing view as base for a new view, instead of always starting afresh
- Mark specific elements as *mandatory*
- Add additional information/instruction for form fields
- Configure to get form submission updates over mail

### How to Create, customize & share?
![Form](https://user-images.githubusercontent.com/86527202/138726122-85ff583c-4a62-464e-99b0-c6b24f23d037.gif)

- Team NocoDB

**Comment by candideu** [comment]
I love this feature! I hope it will be possible to upload our own logos and background images for the forms, as well as being able to choose different colour schemes.

**Reply by dstala** [reply]
@candideu Right now, we do not support background images/ logos on Form. Created an enhancement request.

https://github.com/nocodb/nocodb/issues/1291
Feel free to add more details to this request.

**Reply by candideu** [reply]
Thank you!

**Comment by wfdigitalsupport** [comment]
Great feature team. 

Any plans to add basic field validation on the form builder? Maybe supporting using regex
